user/views.py

```diff
@@ -22,12 +22,51 @@
 from django.conf import settings
 
 API_KEY = os.getenv('API_KEY')
+TMDB_API_KEY = os.getenv('TMDB_API_KEY')
 
 @login_required
 def shows(request):
     shows = Show.objects.all().order_by('-time')
     return render(request, 'shows.html', {'shows': shows})
 
+@login_required
+def movie_details(request):
+    """
+    DISPLAY A PAGE WITH DETAILS ABOUT THE MOVIE ON WHICH THE USER CLICKS
+    ALSO SHOW LIST OF CAST MEMBERS
+    """
+
+    id=request.GET.get('imdb_id','')
+    movie=Movies.objects.filter(imdb_id=id)
+    if movie:
+        
+        # FETCH THE CAST LIST OF THE MOVIE
+        tmdb_api_key=TMDB_API_KEY
+        url=f"https://api.themoviedb.org/3/movie/{id}/credits?api_key={tmdb_api_key}"
+        response=requests.get(url)
+        filtered_cast={}
+        if response.status_code == 200:
+            data=response.json()
+            croplen=0
+            if len(data['cast'])>=10:
+                croplen=10
+            else:
+                croplen=len(data['cast'])
+            filtered_cast = [
+                {
+                    'name': member['name'],
+                    'character': member['character'],
+                    'img_url' : f'https://image.tmdb.org/t/p/w200{member['profile_path']}'
```

Comment:
I am getting the following error 
`'img_url' : f'https://image.tmdb.org/t/p/w200{member['profile_path']}'
                                                          ^^^^^^^^^^^^
SyntaxError: f-string: unmatched '['`
could you change it to?
` 'img_url' : f"https://image.tmdb.org/t/p/w200{member['profile_path']}"`